test/tableSTAT.js

```diff
@@ -0,0 +1,707 @@
+  0x00, 0x00,                       // 000E [1] uint16  axisOrdering
+                              // 0024         Offset  axisValueOffsets[6]
+  0x00, 0x0c,                       // 0000   axisValueOffsets[0]
+  0x00, 0x18,                       // 0002   axisValueOffsets[1]
+  0x00, 0x24,                       // 0004   axisValueOffsets[2]
+  0x00, 0x30,                       // 0006   axisValueOffsets[3]
+  0x00, 0x3c,                       // 0008   axisValueOffsets[4]
+  0x00, 0x48,                       // 000A   axisValueOffsets[5]
+                              // 0030  000C   axisValue[0]
+  0x00, 0x01,                       // 0000     uint16  format
+  0x00, 0x00,                       // 0002     uint16  axisIndex
+  0x00, 0x02,                       // 0004     uint16  flags
+  0x01, 0x03,                       // 0006     uint16  valueNameID
+  0x00, 0x00, 0x00, 0x00,           // 0008     Fixed(16.16)  value  0.0
+                              // 003C  0018   axisValue[1]
+  0x00, 0x01,                       // 0000     uint16  format
+  0x00, 0x01,                       // 0002     uint16  axisIndex
+  0x00, 0x00,                       // 0004     uint16  flags
+  0x01, 0x01,                       // 0006     uint16  valueNameID
+  0x01, 0x2c, 0x00, 0x00,           // 0008     Fixed(16.16)  value  300.0
+                              // 0048  0024   axisValue[2]
+  0x00, 0x01,                       // 0000     uint16  format
+  0x00, 0x01,                       // 0002     uint16  axisIndex
+  0x00, 0x00,                       // 0004     uint16  flags
+  0x01, 0x02,                       // 0006     uint16  valueNameID
+  0x01, 0x5e, 0x00, 0x00,           // 0008     Fixed(16.16)  value  350.0
+                              // 0054  0030   axisValue[3]
+  0x00, 0x01,                       // 0000     uint16  format
+  0x00, 0x01,                       // 0002     uint16  axisIndex
+  0x00, 0x00,                       // 0004     uint16  flags
+  0x01, 0x03,                       // 0006     uint16  valueNameID
+  0x01, 0x90, 0x00, 0x00,           // 0008     Fixed(16.16)  value  400.0
+                              // 0060  003C   axisValue[4]
+  0x00, 0x01,                       // 0000     uint16  format
+  0x00, 0x01,                       // 0002     uint16  axisIndex
+  0x00, 0x00,                       // 0004     uint16  flags
+  0x01, 0x04,                       // 0006     uint16  valueNameID
+  0x02, 0x58, 0x00, 0x00,           // 0008     Fixed(16.16)  value  600.0
+                              // 006C  0048   axisValue[5]
+  0x00, 0x01,                       // 0000     uint16  format
+  0x00, 0x01,                       // 0002     uint16  axisIndex
+  0x00, 0x00,                       // 0004     uint16  flags
+  0x01, 0x05,                       // 0006     uint16  valueNameID
+  0x02, 0xbc, 0x00, 0x00            // 0008     Fixed(16.16)  value  700.0
+];                                  // 0078
```

Comment:
As mentioned in change message and test file, checking even the few example font files I could find with STAT table, found differing data formats.  Interestingly the deprecated/undocumented STAT format 1.0 was found twice.  My intent with the sets of binary/expanded data was to exercise the format variations found so far, including missing or present (non-null) data.  Otherwise the older format 0 file would have been a surprise requiring a fix.

I was thinking including the self-contained binary data in the test was better than committing to including multiple font files.  And certainly thinking that testing variations is better than not.

If `Selawik-variable.ttf` is included as proposed here, then one of the binary stanzas can go away in favor of reading from the file directly.  (I picked this font file to include because of all the other newer tables it also contains)

The `TestGVAREight.ttf` is from the same series of test file you have already sampled, and is quite small.  We could switch to include it here and skip that binary table.

Questions:
* multiple tests where variations seen?  (formats, data presence/absence, etc.)
* self-contained data for multiple testing where sample fonts are otherwise uninteresting?
* what guidance on when to include new font files to drive testing?

Notes:
> 481,504 Selawik-variable.ttf
> 106,184 AmstelvarAlpha-VF.ttf
>  264,380 Decovar-VF.ttf
> &nbsp;&nbsp;&nbsp;  4,692 TestGVAREight.ttf     (from text-rendering-tests)